### PR TITLE
🔧 Allow global setting of prefix commands options

### PR DESCRIPTION
**Pull request by celianvdb**
Hi!
This PR just add the possibility to set options of prefix commands globally.
Have a good day

### CODE REVIEW

**Comment by celianvdb** [comment]
Renamed as DRAFT since I maybe found a bug due to this change.

File: packages/core/src/definitions/interfaces/discord-module-options.ts [hunk 1]
```diff
@@ -49,6 +50,12 @@ export interface DiscordModuleOption {
    */
   prefix?: string;
 
+  /**
+   * Global options for prefix command
+   */
+
```
Comment (fjodor-rybakov) [inline packages/core/src/definitions/interfaces/discord-module-options.ts:56]:
```suggestion
```

File: packages/core/src/definitions/interfaces/prefix-command-global-options.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+export interface PrefixCommandGlobalOptions {
```
Comment (fjodor-rybakov) [inline packages/core/src/definitions/interfaces/prefix-command-global-options.ts:1]: I think this needs to be added to index file too [here](https://github.com/fjodor-rybakov/discord-nestjs/blob/master/packages/core/src/index.ts#L3)

**Review by fjodor-rybakov — CHANGES_REQUESTED** [review]

**Comment by fjodor-rybakov** [comment]
LGFM

**Pull request merged**